db/migrate/20180912181112_campaigns_genras.rb

```diff
@@ -0,0 +1,6 @@
+class CampaignsGenras < ActiveRecord::Migration[5.2]
```

Comment:
Does it matter that it states 'CampaignGenras' and not 'CampaignGenres'.
a vs e?

Reply:
No, but I’ll correct that. Good spot